src/game.cpp

```diff
@@ -1385,6 +1385,8 @@ bool Game::createClient(const std::string &playername,
 	mapper = client->getMinimap();
 	if (mapper)
 		mapper->setMinimapMode(MINIMAP_MODE_OFF);
+	
+	input->setScript(client->getScript());
```

Comment:
This will turn into garbage when the client exits to mainmenu. Please set this value to `nullptr` on `Game::shutdown`